src/bin/pg_autoctl/cli_create_node.c

```diff
@@ -91,7 +91,8 @@ CommandLine create_postgres_command =
 		"  --pg-hba-lan      edit pg_hba.conf rules for --dbname in detected LAN\n"
 		KEEPER_CLI_SSL_OPTIONS
 		"  --candidate-priority    priority of the node to be promoted to become primary\n"
-		"  --replication-quorum    true if node participates in write quorum\n",
+		"  --replication-quorum    true if node participates in write quorum\n"
+		"  --maximum-backup-rate	maximum transfer rate of data transferred from the server during initial sync\n",
```

Comment:
Nitpick: please review alignment of the help strings, "maximum" should start on the same column as "true" one line above.

Reply:
Sure, sorry looks like I added an extra tab there.